CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Changes
 * Move minimum node version to 8.x due to upstream packages using newer ES features.
   (`#2707 <https://github.com/girder/girder/pull/2707>`_).
 
+Bug Fixes
+---------
+
+Python Client
+^^^^^^^^^^^^^
+
+* The python client now supports uploading empty files.
```

Comment:
To avoid repeating the section title, this could read "Support uploading empty files."